static/bidder-info/imds.yaml

```diff
@@ -13,8 +13,10 @@ capabilities:
 userSync:
   supportCors: true
   iframe:
-    url: "https://ad-cdn.technoratimedia.com/html/usersync.html?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}"
+    url: "https://ad-cdn.technoratimedia.com/html/usersync.html?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gppsid={{.GPPSID}}&cb={{.RedirectURL}}"
     userMacro: "[USER_ID]"
   redirect:
-    url: "https://sync.technoratimedia.com/services?srv=cs&gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}"
+    url: "https://sync.technoratimedia.com/services?srv=cs&gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gppsid={{.GPPSID}}&cb={{.RedirectURL}}"
```

Comment:
@ecammit  `uid` was not correctly substituted in redirect url

`https://ib.adnxs.com/prebid/setuid?bidder=imds&gdpr=&gdpr_consent=&f=i&uid=GDPR`

Reply:
> @ecammit `uid` was not correctly substituted in redirect url
> 
> `https://ib.adnxs.com/prebid/setuid?bidder=imds&gdpr=&gdpr_consent=&f=i&uid=GDPR`

We only support the US and Canada today. Please test from an IP located in either country. This works as expected.

Reply:
@VeronikaSolovei9 will check this out.

Reply:
To be clear: The value of the `uid` has no relation to this pull request. If there is a problem that you'd like to report regarding how our sync endpoint is passing the `uid` value, please email the maintainer email for the adapter (eng-demand@imds.tv).

Reply:
I was able to successfully test the redirect and iframe urls.